src/lib/Transport/index.ts

```diff
@@ -140,6 +127,22 @@ export default class Transport {
       });
   }
 
+  private prepareHeaders(headers: IncomingHttpHeaders, context: string): IncomingHttpHeaders {
+    const referer = (headers.referer.indexOf('?') >= 0)
+      ? (`${headers.referer}&context=${context}`)
+      : (`${headers.referer}?userId=27&customer=dev&locale=en-US&platform=desktop&context=${context}`);
+
+    const newHeaders = {
+      ...headers,
+      ...this.client.getAuthHeader(),
+      referer,
+    };
+
+    delete newHeaders.host;
```

Comment:
I think it would be good to keep a comment here of why we are deleting this header. Also, would doing this work?
```
   const newHeaders = {
      ...headers,
       ...this.client.getAuthHeader(),
       referer,
       host: undefined,
     };
```

Reply:
Seems to work from what I just tested. I like that better.